src/components/common/BannerMessage.svelte

```diff
@@ -30,12 +34,16 @@
   .warning {
     background-color: var(--brand-alert-yellow);
   }
+  .default {
+    border-bottom: 1px solid var(--border-color);
+    background-color: var(--background-content);
+  }
   span {
     font-weight: bold;
     margin-right: calc(var(--spacer) / 12);
   }
   p {
-    margin: 0;
+    margin-bottom: 0;
     font-weight: 500;
     font-size: var(--font-size-mini);
```

Comment:
I'd keep the font size of the banner the same as the market's: `font-size: var(--font-size-small);`